db/institutions.yml

```diff
@@ -1492,3 +1492,252 @@ zhejiangu:
   updated: 2020-02-14 19:37:28.925621
   uuid: 4f88b3c9-4719-4a0e-836b-16af6ab6605f
   year: 2020
+lhmfcas:
+  aka:
+    - Laboratory of High Magnetic Field of Chinese Academy of Sciences
+    - lhmfcas
+  city: Hefei
+  country: China
+  day: 15
+  month: 3
+  name: Laboratory of High Magnetic Field of Chinese Academy of Sciences
+  year: 2020
+lab49:
+  aka:
+    - lab49
+  city: Dublin
+  country: Ireland
+  day: 15
+  month: 3
+  name: Lab49
+  year: 2020
+kyotou:
+  aka:
+    - kyotou
+  city: Kyoto
+  country: Japan
+  day: 15
+  month: 3
+  name: Kyoto University
+  year: 2020
+kenyattau:
+  aka:
+    - kenyattau
+  city: Kahawa
+  country: Kenya
+  day: 15
+  month: 3
+  name: Kenyatta University
+  year: 2020
+icmm:
+  aka:
+    - Instituto de Ciencia de Materiales de Madrid
+    - icmm
+  city: Madrid
+  country: Spain
+  day: 15
+  month: 3
+  name: Instituto de Ciencia de Materiales de Madrid
+  year: 2020
+ideas:
+  aka:
+    - ideas
+    - Institute for the Development of Energy for African Sustainability
+  city: Pretoria
+  country: South Africa
+  day: 15
+  month: 3
+  name: Instituto de Ciencia de Materiales de Madrid
+  year: 2020
+icl:
+  aka:
+    - icl
+    - Imperial College of London
```

Comment:
I think it is usually  just Imperial College.  I would leave it like this, and also "Imperial College, London" in the AKA but the name should maybe be just Imperial College